crates/nu-scroll/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+authors = ["The Nushell Project Developers"]
+description = "Nushell table printing"
+repository = "https://github.com/nushell/nushell/tree/main/crates/nu-scroll"
+edition = "2021"
+license = "MIT"
+name = "nu-scroll"
+version = "0.71.1"
+
+[dependencies]
+nu-ansi-term = "0.46.0"
+nu-protocol = { path = "../nu-protocol", version = "0.71.1" }
+nu-cli = { path = "../nu-cli", version = "0.71.1" }
+nu-color-config = { path = "../nu-color-config", version = "0.71.1" }
+nu-engine = { path = "../nu-engine", version = "0.71.1" }
+nu-table = { path = "../nu-table", version = "0.71.1" }
+
+terminal_size = "0.2.1"
+strip-ansi-escapes = "0.1.1"
+reedline = { version = "0.13.0", features = ["bashisms", "sqlite"] }
+crossterm = "0.24.0"
+tui = "0.19.0"
```

Comment:
Sadly we are still behind the most recent version `0.25` of `crossterm` (haven't gotten around to the necessary changes in `reedline`). `tui` is already on the most recent version, so as it stands this introduces dependency duplication.

Reply:
scroll actually doesn't use reedline I believe.
But `nu-cli` does.

I guess to mitigate this issue we could downgrade tui version, right?

Reply:
Funny thing is that a previous version of `tui` uses `crossterm` `0.23`